demisto_sdk/dev_tools/runner.py

```diff
@@ -0,0 +1,136 @@
+import re
+
+from demisto_sdk.common.configuration import Configuration
+from demisto_sdk.common.tools import print_error, print_color, LOG_COLORS, print_v, print_warning
+from demisto_sdk.common.rest_api import Client, DEMISTO_API_KEY_ENV
+
+
+SECTIONS_REGEX = re.compile(r'^(Context Outputs|Human Readable section|Raw Response section)')
+FULL_LOG_REGEX = re.compile(r'.*Full Integration Log')
+
+
+class Runner:
```

Comment:
love this runner idea, but i think that maybe printing the whole debug log file will be heavy on the terminal and not too readable for the users, what do you think about having an option to save this file into the current dir, and then the user will be able to use that as he pleases

Reply:
Added an argument to `-D` to specify a file name to dump the debug log into.